packages/bigtable/src/table.js

```diff
@@ -436,37 +437,38 @@ Table.prototype.getMetadata = function(options, callback) {
  * @param {object[]} options.ranges - A list of key ranges.
  * @param {module:bigtable/filter} options.filter - Row filters allow you to
  *     both make advanced queries and format how the data is returned.
- * @param {boolean} options.interleave - Allow for interleaving.
  * @param {number} options.limit - Maximum number of rows to be returned.
- * @param {function=} callback - The callback function.
- * @param {?error} callback.err - An error returned while making this request.
- * @param {module:bigtable/row[]} callback.rows - List of Row objects.
+ * @return {stream}
  *
  * @example
+ * table.getRowsStream()
```

Comment:
Since we're switching bigquery.table.getRows() to bigquery.table.createReadStream(), should we switch this to table.createReadStream()?